src/util/store-util.js

```diff
@@ -79,18 +84,17 @@ export function getTransferAssetsFrom (transactions, accountId, settlements = []
     *
     * To avoid it, we uniq the transactions.
     */
-  return _(transformed)
-    .chain()
-    .uniqWith(_.isEqual)
-    .sortBy('date')
-    .reverse()
-    .value()
+  return flow(
```

Comment:
(no need to fix)

We have introduced FP-style partially and I wonder that `fp.compose` might be better than `fp.flow` in that `fp.compose` can provide a right-to-left look like other lines where lodash-fp functions are used. I have no idea which is better though. What do you think? 🤔

Reply:
As I read in article that was mentioned in PR - [Why using `_.chain` is a mistake.](https://medium.com/making-internets/why-using-chain-is-a-mistake-9bc1f80d51ba)

>Argument Ordering
The arguments seem to be reversed because that applies the functions in the computationally correct order. The original code did a map first, then a flatten and then a sort. Remember how compose works and you’ll see that is exactly what’s happening:
>``` javascript
>compose(
>  sortBy(x => x), 
>  flatten, 
>  map(x => [x, x*2])
>);
>```
>Is the same as:
>``` javascript
>sortBy(x => x)(
>  flatten(
>    map(x => [x, x*2])
>  )
>)([1, 2, 3]);
>```
>Where the map is computed first, then passed to flatten and then finally passed to sort. However, because this can be confusing, lodash provides an inverted composition mechanism called flow which allows for ordering in the opposite (arguably more natural) direction:
>``` javascript
>...
>flow(
>  map(x => [x, x*2]), 
>  flatten, 
>  sortBy(x => x)
>)([1, 2, 3]);
>```

In my opinion it's more easier to read code from top to down, so I vote for `flow` version. What do you think @laSinteZ ?

Reply:
I like the `flow` approach since it is very similar to proposed "pipeline" operator https://github.com/tc39/proposal-pipeline-operator and yes, it shows how data is "transformed", step by step.